fontir/src/ir.rs

```diff
@@ -274,14 +278,17 @@ impl From<KernGroup> for KernSide {
     }
 }
 
+#[derive(Default, Debug)]
+pub struct GlobalMetricsBuilder(
+    HashMap<GlobalMetric, HashMap<NormalizedLocation, OrderedFloat<f64>>>,
+);
+
 /// Global metrics. Ascender/descender, cap height, etc.
 ///
 /// Represents the values of these metrics at a specific position in design space.
 /// At a minimum should be defined at the default location.
-#[derive(Serialize, Deserialize, Default, Debug, Clone, PartialEq, Eq)]
-pub struct GlobalMetrics(
-    pub(crate) HashMap<GlobalMetric, HashMap<NormalizedLocation, OrderedFloat<f64>>>,
-);
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq)]
+pub struct GlobalMetrics(HashMap<GlobalMetric, GlobalMetricValues>);
```

Comment:
Can we add docs to the effect that "This type is constructed with a [`GlobalMetricsBuilder`]"?